platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformBuildOperationIntegrationTest.groovy

```diff
@@ -1363,12 +1363,12 @@ class ArtifactTransformBuildOperationIntegrationTest extends AbstractIntegration
         outputContains("Task-only execution plan: [PlannedTask('Task :consumer:hello', deps=[])]")
 
         getPlannedNodes(0)
-        getExecutePlannedStepOperations(0).empty
+        getExecutePlannedStepOperations(1)
 
-        buildOperations.progress(IdentifyTransformExecutionProgressDetails).size() == 2
-        buildOperations.all(ExecuteWorkBuildOperationType).size() == 2
-        buildOperations.all(SnapshotTransformInputsBuildOperationType).size() == 2
-        buildOperations.all(ExecuteTransformActionBuildOperationType).size() == 2
+        buildOperations.progress(IdentifyTransformExecutionProgressDetails).size() == 4
+        buildOperations.all(ExecuteWorkBuildOperationType).size() == 4
+        buildOperations.all(SnapshotTransformInputsBuildOperationType).size() == 4
+        buildOperations.all(ExecuteTransformActionBuildOperationType).size() == 4
```

Comment:
🤔 Why did these increase? Would be nice to have some comments here explaining each count.